src/project-css/app/Jobs/OptimizeSearchIndex.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Http\Request;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+use Illuminate\Support\Facades\DB;
+use Illuminate\Foundation\Bus\DispatchesJobs;
+
+use App\Jobs\OptimizeSearch;
+
+use Log;
+
+class OptimizeSearchIndex implements ShouldQueue
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
+
+    private $tblNme;  
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct($tblNme)
+    {
+        $this->tblNme = $tblNme;
+    }
+
+    /**
+     * Execute the job.
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.